aws/challenges/Backwards/challenge.yml

```diff
@@ -35,7 +35,7 @@ hints:
 - |-
   If your stuck, try running the permissions command in cloudfox and grepping for the word secret:
 
-  `cloudfox aws -p cloudfoxable permissions -v2 | grep -i secret`
+  `cloudfox aws -p cloudfoxable aws permissions -v2 | grep -i secret`
```

Comment:
Thanks for contributing! Did you mean to insert this 'aws' here? I think this one is not necessary.